src/editor/JSTTextView.m

```diff
@@ -65,6 +70,38 @@ - (void)dealloc {
     
 }
 
+- (NSDictionary<NSString*, NSColor*>*)colors {
```

Comment:
Wouldn't it be nice if these were read in from a plist (or json) 🤔

Reply:
That'd be nice indeed.
I suppose I'd need to cache the values on a class property to avoid IO everytime the view appears?

Reply:
I don't think you need to, I don't think it'll be noticeable. We're already hitting the disk to load the xib.